keycloak/realm.go

```diff
@@ -65,8 +66,19 @@ type Realm struct {
 	SupportLocales              []string `json:"supportedLocales"`
 	DefaultLocale               string   `json:"defaultLocale"`
 
-	//extra attributes of a realm, contains security defenses browser headers and brute force detection parameters(those still nee to be added)
-	Attributes Attributes `json:"attributes,omitempty"`
+	//extra attributes of a realm
+	Attributes map[string]interface{} `json:"attributes"`
+
+	BrowserSecurityHeaders BrowserSecurityHeaders `json:"browserSecurityHeaders"`
```

Comment:
Is this a breaking change in Keycloak's API?  It looks like we were using `_browser_header` before - is that going to break this provider's compatibility with older versions of Keycloak?

Reply:
The Realm tests stills work if I run on ac test with 'make local' and docker-compose pointing to keycloak version 4.8.3.Final
There 2 ways of setting these parameters, The UI is using the way above
I have no idea why there are 2 ways to do this. If you set both ways at the same time with different values, then setting it directly as a real parameters wins from setting it through the attributes.
Thus this seems to be the safer bet and a more correct way to do it through this method.